README.md

```diff
@@ -8,93 +8,46 @@ QuestCraft, utilizing Vivecraft and Pojlib, is a standalone port of Minecraft: J
-
 First off, you most likely want to install QuestCraft. Please go to the release page for **[QuestCraft](https://github.com/QuestCraftPlusPlus/QuestCraft/releases/latest)** and download the APK to your computer or Android device.
 
 Secondly, you must install the QuestCraft APK to your VR Headset using **[SideQuest](https://sidequestvr.com/setup-howto)** and follow the necessary steps to connect your Quest. Drag and drop the APK, and wait for it to install.
 
 Next, open your headset's QuestCraft launcher (through Unknown Sources). Be sure to grant all requested permissions. Once you're done, you'll follow the on-screen prompts to enter your Minecraft login details, then press play and wait for all of the files to download (aproxx. 10 minutes).  (You must do this for each specific version to play that version!)
 
-**NOTE: You may encounter crashes when downloading your selected MC version, ESPECIALLY on v53 and later. This is because Meta can't fix memory leaks, for some reason. Don't fret; launch the app again, press play again, and the download will continue where it left of before the crash. If it continues, please keep trying. If, after 4 attempts, you still are not loaded in, please visit the support channel of the QuestCraft Discord (discord.gg/questcraft)**
+**NOTE: You may encounter crashes when downloading your selected MC version, ESPECIALLY on v53 and later. This is because Meta can't fix memory leaks, for some reason. Don't fret; launch the app again, press play again, and the download will continue where it left of before the crash. If it continues, please keep trying. If, after 4 attempts, you still are not loaded in, please visit the support channel of the [QuestCraft Discord](https://discord.gg/questcraft)**
 
-And boom! You have successfully installed QuestCraft on your VR headset! To play, press the **Play** button and wait for it to load; depending on the headset's age, it might take a while.
+And boom! You have successfully installed QuestCraft on your VR headset! To play, press the **Play** button and wait for it to load; depending on the headset, it might take a while.
 
-## Supported Renderers (Take note when installing mods!)
+## Supported Renderer
 
-- ### Zink
-**[Zink](https://docs.mesa3d.org/drivers/zink.html)**, works with many mods, at the cost of performance. (which is mostly fixed with the performance mods below)
-- ### Vulkan
-**[Vulkan](https://www.vulkan.org/)**, while very fast, it may only works with a small amount of mods. For a list, please see the pinned discussion in #discussion on the **[VulkanMod Server](https://discord.gg/FVXg7AYR2Q)**. VulkanMod is not affiliated with us, so please don't ask and spam the devs questions related to QuestCraft! 
-*Vulkan is not yet out for QuestCraft, please wait!*
+**Regal**
+We use an renderer named Regal, a almost perfect renderer for Minecraft. You may find issues, you can see some renderer specific issues on our Discord, and in the known-issues-5•0
```

Comment:
Bruh English really is confusing. But in case you forgot, you only use the word "an" instead of "a" if the word after it starts with a vowel. So instead of "we use an renderer named Regal" it would be "we use a renderer named regal". Same goes for "a almost" --> "an almost"

Reply:
Also when separating independent clauses in the sentence "You may find issues, you can see some renderer specific issues on our Discord, and in the known-issues-5.0" it would technically be written like this: "You may find issues; you can see some renderer specific issues on our Discord and in the known-issues-5.0" with a semi-colon splitting the clauses.
🤓 🤓 🤓 
(nah i just playing lol)

Reply:
> Also when separating independent clauses in the sentence "You may find issues, you can see some renderer specific issues on our Discord, and in the known-issues-5.0" it would technically be written like this: "You may find issues; you can see some renderer specific issues on our Discord and in the known-issues-5.0" with a semi-colon splitting the clauses. 🤓 🤓 🤓 (nah i just playing lol)

lol i put this exact thing in my grammatically correct readme file on my fork :3